appveyor.yml

```diff
@@ -0,0 +1,23 @@
+# Test against the latest version of this Node.js version
+environment:
+  nodejs_version: "6"
```

Comment:
Does appveyor support testing on node versions 6 and version 8?

Reply:
Yes it support `build matrix` i've updated config file